lib/fathead/mdn_css/parse.pl

```diff
@@ -441,10 +441,15 @@ sub parse_initial_value {
                 #get text not in ul
                 $initial_value .= $td->at('ul')->remove->all_text;
 
-                #add new line after each li text so that it appears correctly
-                for my $li ( $ul->find('li')->each ) {
-                    $initial_value .= $li->all_text . '\\n ';
+                my $li_collection = $ul->find('li');
+                my $last_li       = $li_collection->last;
+                $li_collection->last->remove;
+
+                #Separate multiple values by commas
+                for my $li ( $li_collection->each ) {
```

Comment:
The result could/should look something like this...

![border-top_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/17793913/f68f450a-6578-11e6-8fa5-bab8f1aa8080.png)